man/nvim.1

```diff
@@ -373,8 +373,24 @@ Used to set the 'shell' option, which determines the shell used by the
 command.
 .It Ev NVIM_TUI_ENABLE_CURSOR_SHAPE
 If defined, change the cursor shape to a vertical bar while in insert mode.
-Requires that the host terminal support the DECSCUSR CSI escape sequence.
+Requires that the host terminal supports the DECSCUSR CSI escape sequence.
 Has no effect in GUIs.
+.Pp
+Depending on the terminal emulator, using this option with
```

Comment:
"with under"? I'm not up to speed with nroff macros, so perhaps there is something I'm not seeing here, but this sounds weird at least.